cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -54,7 +54,7 @@ msgstr "Ydych chi mewn perygl uniongyrchol o niwed?"
 msgid "Have you already started family mediation? (This includes cases that have already finished)"
 msgstr "Ydych chi wedi dechrau cyfryngu teuluol eisoes? (Mae hyn yn cynnwys achosion sydd wedi dod i ben eisoes)"
 msgid "Have you taken part in a family mediation session?"
-msgstr "Ydych chi wedi cymryd rhan mewn sesiwn gyfryngu i’r teulu?"
+msgstr "Ydych chi wedi cymryd rhan mewn sesiwn cyfryngu i deuluoedd?"
```

Comment:
Will the english not also need to change for this ?

Reply:
No, the Welsh we had was a placeholder from a different context.  Once we got the translation, we just updated the Welsh.  No change to English.